### DISCUSSION TITLE
[Bug] git install Enabling experimental support for pseudo consoles"

### DISCUSSION THREAD

**Discussion by Joredjs** [ScoopInstaller/Main#3536] [Q&A] [answered]
## Bug Report

I cant login in gh with git-bash, both installed with scoop

### Current Behaviour

* in cmd:

```
$ scoop install gh git
```
* in git-bash:

```
$ gh auth login

?What account do you want to log into? [Use arrows to move, type to filter]
>GitHub.com
GitHub Enterprise Server

could not prompt: Incorrect function.
```

### Expected Behaviour

When i try to login with gh, not errors message appear , but instead i can loggin 

```
$ gh auth login

?What account do you want to log into? [Use arrows to move, type to filter]
>GitHub.com
GitHub Enterprise Server
```

### Additional context/output

Searching in the official site of gh [link](https://cli.github.com/manual/gh_help_mintty) they recommend *Reinstall Git for Windows, checking "Enable experimental support for pseudo consoles"*. I tried to re install git with scoop and i dont know how check this option or sending what flag.

### Possible Solution

Exploring the files in my folder scoop i noticed that if exectute the  ***scoop/apps/git/current/bin/bash.exe*** file it works perfect, with no errors, but when i execute the ***scoop/apps/git/current/git-bash.exe*** file the error appears. Maybe in the shim creation, pointing to this file, this error could be  solved

### System details

**Windows version:** Windows 11 Home

**OS architecture:** 64 bits x64

**PowerShell version:** 5.1.19041.1682

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-04-29T14:12:50.9759531-05:00",
    "SCOOP_REPO":  "https://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master"
}
```

**Comment by rashil2000** [comment, marked as answer]
Can you create an environment variable MSYS with the value enable_pcon and log out and try again?

**Comment by Joredjs** [comment]
Thank you @rashil2000, it works perfectly !!
